orga/tests/acceptance/login-test.js

```diff
@@ -21,146 +21,67 @@ module('Acceptance | Login', function (hooks) {
     sinon.stub(domainService, 'getExtension');
   });
 
-  module('When usePixOrgaNewAuthDesign is false', function (hooks) {
-    hooks.beforeEach(function () {
-      server.create('feature-toggle', {
-        id: 0,
-        usePixOrgaNewAuthDesign: false,
-      });
-    });
-
-    test('renders legacy view', async function (assert) {
-      // given & when
-      const screen = await visit('/connexion');
+  test('renders new layout', async function (assert) {
```

Comment:
Avec cette PR il n’y a plus de notion d’ancien et de nouveau layont : 
```suggestion
  test('renders layout', async function (assert) {
```